src/pages/ContactPage.vue

```diff
@@ -1,6 +1,36 @@
 <template>
   <div>
     <h1>Contact Us</h1>
+    <div class="contact-page">
+      <div class="contact-page__image">
+        <img src="@/assets/img/map.png" alt="map" />
+      </div>
+      <div class="contact-page__address">
+        <img src="@/assets/img/pinlocation.svg" alt="location pin icon" />
+        <p>
+          Monterail Sp. z o.o. <br />
+          ul. Oławska 27-29 <br />
+          50-123 Wrocław <br />
+          Poland
+        </p>
+        <img src="@/assets/img/emailicon.svg" alt="email icon" />
+        <p>hello@monterail.com</p>
+        <img src="@/assets/img/phoneicon.svg" alt="phone icon" />
+        <p>+48 533 600 136</p>
+      </div>
```

Comment:
I think this part is duplicated in `ContactUs` component.
We could try to refactor this so we could re-use this part in both places - what do you think?

Reply:
Valid point taken :D Thank you 🥇